astropy/convolution/tests/test_kernel_class.py

```diff
@@ -622,22 +635,11 @@ def test_kernel2d_initialization(self):
         with pytest.raises(TypeError):
             Kernel2D()
 
-    def test_array_keyword_not_allowed(self):
+    @pytest.mark.parametrize(["kernel", "opt"], KERNEL_TYPES_WITH_ARGS)
+    def test_array_keyword_not_allowed(self, kernel, opt):
         """
         Regression test for issue #10439
         """
         x = np.ones([10, 10])
         with pytest.raises(TypeError, match=r".* allowed .*"):
```

Comment:
```suggestion
        with pytest.raises(TypeError, match=r"Array argument not allowed for kernel.*"):
```
seems GitHub is not keeping up with current code version at all today; but if this is still valid, might as well put a fuller message here for clarity.